receiver/postgresqlreceiver/documentation.md

```diff
@@ -82,7 +82,7 @@ Number of times the background writer stopped a cleaning scan because it had wri
 
 | Unit | Metric Type | Value Type | Aggregation Temporality | Monotonic |
 | ---- | ----------- | ---------- | ----------------------- | --------- |
-|  | Sum | Int | Cumulative | true |
+| {stoppages} | Sum | Int | Cumulative | true |
```

Comment:
Technically I think you could describe it this way, but in my opinion this is a bit much as "stoppages" aren't really a native concept in postgresql.

Reply:
What do you think about using `{events}` instead ?

Reply:
@djaglowski, I agree. Maybe `{stop event}` or just `1` which doesn't seem to conflict with https://github.com/open-telemetry/semantic-conventions/blob/beb944465136ff0661d9d0cdb2f5c588830b4998/specification/metrics/semantic_conventions/README.md#instrument-units

BTW looks like unit pluralization is now being discouraged in the spec, so we shouldn't use it anymore.

Reply:
Both make sense to me. `{stop event}` is more explicit -- but this may still be a little forced.

> BTW looks like unit pluralization is now being discouraged in the spec, so we shouldn't use it anymore.

I've updated `{tables}` to `{table}` 👍